spec/views/sign_up/registrations/new.html.slim_spec.rb

```diff
@@ -30,10 +29,10 @@
     expect(rendered).to have_xpath("//form[@autocomplete='off']")
   end
 
-  it 'includes a form to cancel account creation' do
+  it 'includes a link to return to the home page' do
     render
-    link = t('links.cancel_account_creation')
+    link = t('links.cancel')
 
-    expect(rendered).to have_selector("input[value='#{link}']")
+    expect(rendered).to have_content(link)
```

Comment:
What do you think about replacing these 2 last lines with this, to test both the link text and the actual destination?

```ruby
expect(rendered).to have_link(t('links.cancel'), href: root_url)
```
I would expect the cancel link to point to the root url, not `api/saml/logout` since the user is not signed in.